frontend/src/sidebar/Header.svelte

```diff
@@ -73,6 +73,7 @@
     position: absolute;
     top: var(--header-height);
     left: 19px;
+    z-index: 1;
```

Comment:
Please use one of the CSS variables for this z-index (they're all defined in style.css)